code/game/objects/items/religion.dm

```diff
@@ -347,20 +347,18 @@
 	desc = "This armour is said to be based on the armor of kings on another world thousands of years ago, who tended to assassinate, conspire, and plot against everyone who tried to do the same to them.  Some things never change."
 
 
-/obj/item/storage/box/itemset/crusader/blue/New()
-	..()
+/obj/item/storage/box/itemset/crusader/blue/PopulateContents()
+	. = ..()
 	contents = list()
```

Comment:
These two lines (parent call and contents clear) cancel each other out and can both be removed.
It's considered normal for `PopulateContents` not to call parent.
Ditto below.